man/adoc/bpftrace.adoc

```diff
@@ -49,6 +49,18 @@ List all the probes attached in the program::
 
 x86_64, arm64, s390x, arm32, loongarch64, mips64, ppc64, riscv64
 
+== Script Licenses
+
+The license of a BpfScript program affects how it can be run, as the Linux kernel limits the functionality available to non-GPL licensed programs.
```

Comment:
I don't see anywhere else in the docs or code that refers to these programs as "BpfScript". I know we want to go in this direction but maybe for now we should refer to them as "bpftrace programs" or "bpftrace scripts".